geopyspark/geotrellis/render.py

```diff
@@ -1,4 +1,6 @@
 from geopyspark.geotrellis.constants import RESAMPLE_METHODS, NEARESTNEIGHBOR, ZOOM, COLOR_RAMPS
+from .rdd import RDDWrapper
+from pyspark.storagelevel import StorageLevel
```

Comment:
This will need an environment check like `rdd` https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/rdd.py#L8

Because if you try to import this first the way it is now, it'll throw an error since it won't know where to find the `pyspark` library.